connections.md

```diff
@@ -0,0 +1,11 @@
+Connections Game
+================
+
+Another critical element of stories is the connections between characters. Indeed, one role of stories is to connect us who read them.
+
+The Challenge
+-------------
+Take two characters from two different stories, and list:
+* 5 commonalities they share.
```

Comment:
The game should start off easier than this: all people need to connect them in a community, to identify each other as "another self", they only need one connection.